src/parser/jobs/blm/modules/Sharpcast.tsx

```diff
@@ -158,5 +162,22 @@ export class Sharpcast extends Analyser {
 				<Plural value={this.sharpedScathes} one="# Sharpcast was" other="# Sharpcasts were"/> consumed by <DataLink action="SCATHE" />.
 			</Trans>,
 		}))
+
+		// Suggestion not to overwrite Sharpcast
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.SHARPCAST.icon,
+			content: <Trans id="blm.sharpcast.suggestions.overwrote-sharpcasts.content">
+				You lost at least one guaranteed <DataLink status="THUNDERCLOUD" /> or <DataLink status="FIRESTARTER" /> proc by using <DataLink action="SHARPCAST" /> while the status was already active.
+			</Trans>,
+			tiers: {
+				1: SEVERITY.MINOR,
+				3: SEVERITY.MEDIUM,
+				5: SEVERITY.MAJOR,
+			},
```

Comment:
These tiers seem fairly lenient for an active mistake (pressing the button too many times / too soon).  Is the potency gain from this skill small enough to "allow" wasting 2 charges before we complain by default?

Reply:
The tiers were cloned from the dropped Sharpcast version (probably should've made a const for it...). They're fairly lenient since dropping/overwriting Sharpcast might not have any effect, since you could wind up with the Firestarter/Thundercloud that Sharpcast was going to force anyways due to luck.